Sim_To_DuT_Interface/DuTLogger/DuTLogger.h

```diff
@@ -0,0 +1,135 @@
+/**
+ * DuT-Logger
+ *
+ * Contains the class DuTLogger and all necessary variables.
+ * For better documentation see the class "DuTLogger" and it's functions.
+ *
+ * @author Marco Keul
+ * @version 1.0
+ */
+
+#ifndef TESTLOGGER_DUTLOGGER_H
+#define TESTLOGGER_DUTLOGGER_H
+#include <string>
+#include "quill/Quill.h"
+#include <list>
+#include <filesystem>
+
+/**
+ * Defines the relative path where the logfiles will be created.
+ * In this path the log messages will be logged.
+ */
+static const std::string PATH_CONSOLE_LOG = "/logs/console";
+
+/**
+ * Defines the relative path where the logfiles will be created.
+ * In this path the data objects will be logged.
+ */
+static const std::string PATH_DATA_LOG = "/logs/data";
+
+/**
+ * This constant defines the mode how the logger will open the logfile of the console logging.
+ * Mode "w" creates a new file or overwrite the existing file at every start of the application.
+ * In mode "a" the logger tries to append it's content on an existing file.
+ */
+static const std::string FILE_MODE_CONSOLE = "a";
+/**
+ * This constant defines the mode how the logger will open the logfile of the data logging.
+ * Mode "w" creates a new file or overwrite the existing file at every start of the application.
+ * In mode "a" the logger tries to append it's content on an existing file.
+ */
+static const std::string FILE_MODE_DATA = "a";
+
+/**
+ * This constant defines the maximum number of files in a logging directory. If there are more
+ * logfiles under the underlying path, old ones will be deleted. (like a date based file rotation)
```

Comment:
Suggestion: "parth, old ones" -> "path that the constant permits, the oldest logfiles will be deleted."